apollo-router/src/services/supergraph_service.rs

```diff
@@ -1451,7 +1451,6 @@ mod tests {
             .unwrap();
         let mut stream = service.oneshot(request).await.unwrap();
         let res = stream.next_response().await.unwrap();
-        assert_eq!(&res.data, &Some(serde_json_bytes::Value::Null));
```

Comment:
The following line is `insta::assert_json_snapshot!(res);` which will also fail if that `assert_eq` would fail, but give more useful info (namely, include `res.errors`)